packages/connect-ui/src/views/Transport.tsx

```diff
@@ -1,24 +1,41 @@
-import { Button, Link, Image } from '@trezor/components';
+import { Button, Image } from '@trezor/components';
 import type { UiEvent } from '@trezor/connect';
-import { SUITE_BRIDGE_URL } from '@trezor/urls';
+import { SUITE_URL, SUITE_BRIDGE_DEEPLINK } from '@trezor/urls';
 
 import { View } from '../components/View';
 import imageSrc from '../images/man_with_laptop.svg';
 
 export type TransportEventProps = Extract<UiEvent, { type: 'ui-no_transport' }>;
 
-export const Transport = () => (
-    <View
-        title="Install Bridge"
-        description="The Bridge is a communication tool, which facilitates the connection between
-        your Trezor device and your internet browser."
-        image={<Image imageSrc={imageSrc} />}
-        buttons={
-            <Link variant="nostyle" href={SUITE_BRIDGE_URL} onClick={() => window.closeWindow()}>
-                <Button variant="primary" icon="EXTERNAL_LINK" iconAlignment="right">
-                    Install Bridge
-                </Button>
-            </Link>
-        }
-    />
-);
+export const Transport = () => {
+    const handleOpenSuite = () => {
+        // todo: can we somehow check if the app is installed (not running ) and present a deeplink in that case?
```

Comment:
this is probably not possible I guess :(

Reply:
I've seen this work on mobile. I have some ideas - try the deeplink, wait some period of time, then check via the HTTP server if it's alive. If not go to the download page. The only problem in this case is the necessary delay...

Reply:
🤔 what if I don't have the app registered for the deeplink but it is already installed? handling based on focus? until user focuses this page again, we show some spinner and expected that he is clicking somewhere in native dialogues?

Reply:
Focus sounds like it could be viable. On Mac it seems like the deeplinks work "automagically", not sure about other platforms.